ui/component/wunderbar/view.jsx

```diff
@@ -127,7 +126,14 @@ class WunderBar extends React.PureComponent<Props, State> {
     const { navigateToUri, doSearch, doShowSnackBar } = this.props;
 
     if (suggestion.type === SEARCH_TYPES.SEARCH) {
-      doSearch(query);
```

Comment:
I don't think this needs to change. With this change you can't just type something in and search, it will always try to go to the file page if the search term is a valid uri

Reply:
Thanks, I have pushed the branch without this change.

Reply:
It looks like this code is still here

Reply:
Sorry, it's now gone. Let me know if there are any more comments.